package.json

```diff
@@ -67,6 +67,7 @@
   "scripts": {
     "postinstall": "./node_modules/.bin/rn-nodeify --install buffer,events,process,stream,util,inherits,fs,path --hack; npx jetify",
     "clean": "cd android/; ./gradlew clean; cd ..; rm -r -f /tmp/metro-cache/; rm -r -f node_modules/; yarn cache clean --force; yarn install",
+    "clean:ios": "rm -fr node_modules && rm -fr ios/Pods && npm i && cd ios && pod update && cd ..; npm start -- --reset-cache",
```

Comment:
we use yarn, it should be:
`"clean:ios": "rm -fr node_modules && rm -fr ios/Pods && yarn install && cd ios && pod update && cd ..; yarn start -- --reset-cache",`

Reply:
done